src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -2237,79 +2228,48 @@ FullWidthRepeat2:
             Dim FirstValueStart As Integer = Here
 
             ' // The first thing has to be an integer, although it's not clear what it is yet
-            If Not ScanIntLiteral(FirstValue, Here) Then
-                Return Nothing
-
-            End If
-
+            If Not ScanIntLiteral(FirstValue, Here) Then Return Nothing
```

Comment:
@AdamSpeight2008, please avoid changing block-If statements into line-If statements. You will be asked to undo changes like this every time.